jumpcutter.py

```diff
@@ -12,6 +12,8 @@
 import os
 import time
 
+ffmpegPath = 'ffmpeg'
```

Comment:
Change to naming convention according to pep8
```suggestion
FFMPEG_PATH = 'ffmpeg'
```